web_ir_actions_act_multi/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,4 @@
+* Petar Najman <petar.najman@modoolar.com>
+* Mladen Meseldzija <mladen.meseldzija@modoolar.com>
+* Alexey Pelykh <alexey.pelykh@brainbeanapps.com>
+* Manuel Calero - Tecnativa
```

Comment:
Write your name under a link to the Tecnativa website like it is here https://github.com/OCA/web/blob/13.0/web_decimal_numpad_dot/readme/CONTRIBUTORS.rst